src/services/handleSorting.js

```diff
@@ -29,12 +30,46 @@ const handleSorting = (sortType, page = '') => {
     case 'Name':
       return (a, b) => a.tagname.localeCompare(b.tagname);
     case 'Username':
-      return (a, b) => a.username.localeCompare(b.username);
+      return (a, b) => a.localeCompare(b.username);
```

Comment:
This is breaking in the users page, pls revert

Reply:
ok